src/main/java/org/breedinginsight/brapps/importer/controllers/UploadController.java

```diff
@@ -100,7 +100,7 @@ public HttpResponse<Response<ImportResponse>> getUploadData(@PathVariable UUID p
                 return HttpResponse.ok(response);
             }
         } catch (DoesNotExistException e) {
-            log.info(e.getMessage());
+            log.info(e.getMessage(), e);
```

Comment:
Is there a.reason this exception is logged as info message while the others are logged as error?

Reply:
@HMS17  Good question.  It was originally an `INFO` message, I just changed which method is being called so that the stack trace gets included in the log message.  Happy to change it to `ERROR` if you'd like.

Reply:
Error seems more consistent but don't feel particularly strongly about it, mostly wanted to understand.